metrique/README.md

```diff
@@ -4,9 +4,15 @@ metrique is a crate to emit unit-of-work metrics
 
 ## Getting Started (Applications)
 
-Most metrics your application records will be "unit of work" metrics. These are typically tied to the request/response scope. You declare a struct the represents the metrics you plan to capture over the course of the request and annotate it with `#[metrics]`
+Most metrics your application records will be "unit of work" metrics. These are typically tied to the request/response scope. You declare a struct the represents the metrics you plan to capture over the course of the request and annotate it with `#[metrics]`.
 
-This library exposes a wrapper `<MetricName>Guard` type (a type alias to [`AppendAndCloseOnDrop`]) that implicitly appends to a global queue when the struct is dropped. This wrapper in turn exposes other APIs to further tune behavior.
+The `#[metrics]` macro will make your struct implement the [`CloseEntry`] trait, which represents an entry that
+can be "closed" into an [`InflectableEntry`].
+
+The macro will also expose a wrapper `<MetricName>Guard` type (a type alias to [`AppendAndCloseOnDrop`]) that implicitly appends to a global queue when the struct is dropped. This wrapper in turn exposes other APIs to further tune behavior.
+
+The best way to emit the [`InflectableEntry`] is by defining a global entry sink using the [`metrique_writer::sink::global_entry_sink`] macro, which defines a global variable that provides a global rendezvous point where metrics can be written into. Then, you need to attach to that global entry sink an actual entry sink that will actually write your metrics somewhere - the example below will write the metrics to an [`tracing_appender::rolling::RollingFileAppender]
```

Comment:
the key here is that there is the `global` destination which must be separately wired to a destination. I think if we said something like:

```
Callers can write to the global queue with `::queue()` and wire the queue to a destination with `::attach_to_stream()`.

The queue must be wired to a destination before metrics can be sent to the queue
```